src/evaluators/ConditionalExpression.js

```diff
@@ -48,28 +48,20 @@ export default function(
       return realm.evaluateNodeForEffects(consequent, strictCode, env);
     },
     () => {
-      let completion = env.evaluateCompletion(alternate, strictCode);
+      let completion = env.evaluateCompletionDeref(alternate, strictCode);
```

Comment:
Re:

>On the other hand, conditional expressions are expressions, so perhaps evaluate is the right thing to do here in both cases and we do not need the check for an AbruptCompletion.

I can add `evaluateDeref()` that can't return `AbruptCompletion` if you prefer?